concepts/05 UI Components/DataGrid/30 Filtering and Searching/6 API/5 Filtering Columns Bound to Collection/10 Client-Side Filtering.md

```diff
@@ -0,0 +1,179 @@
+In the code below, you can see how to execute the filtering logic on the client-side. For this purpose, bind the DataGrid to a local array, and disable [remote filtering](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/#filtering).
```

Comment:
> on the client-side.

A hyphen indicates an adjective -- a client-side implementation, a client-side algorithm etc. Right now, we speak just of the client side, as a noun.

```suggestion
The following code example demonstrates how to filter data on the client side. To implement data filtering on the client side, bind the DataGrid to a local array, and disable [remote filtering](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/remoteOperations/#filtering).
```